src/Sidekick.Presentation.Blazor/Overlays/MapInfo/MapInfo.razor

```diff
@@ -0,0 +1,40 @@
+@page "/map/{encodedItem}"
+
+<MudLayout>
+    <SidekickAppBar Title="@Localizer[MapInfoResources.Title]" View="View.Map"></SidekickAppBar>
+    <MudMainContent>
+        <MudContainer Class="py-6">
+            @if (ViewModel.IsSafe)
+            {
+                <MudText Style="color: #00FF00">@Localizer[MapInfoResources.Is_Safe]</MudText>
+            }
+            @foreach (DangerousMapModModel dangerousMod in ViewModel.DangerousMapMods)
+            {
+                string color = $"color: {dangerousMod.Color}";
```

Comment:
I think we should skip that color and use MudBlazor's palette.

Reply:
Sure, that was just how WPF currently does it. Using the MudBlazor palette was how I was hoping we could do this.

Reply:
Replaced styles with MudBlazor Color property